lib/forEach.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function forEach(iterable, callback) {
+  var iterator = iterable[Symbol.iterator]();
```

Comment:
i think this utility should use `Array.from`, and fall back to explicit array/string iteration